### DISCUSSION TITLE
24-bit FLAC

### DISCUSSION THREAD

**Discussion by Nidite** [enzo1982/freac#597] [Help & Support]
Hello!

I'm quite new to all this audio stuff, and I have a very small collection of CDs I've ripped into first WAV files, then converted to FLAC. All my FLACs are in 16-bit format, and so I'd like to re-rip all of them in 24-bit. The way I've found to do this is to enable the Sample Format Converter component in the Processing configuration screen and set the sample resolution to 24. Is this the right way to do it, or is this just converting the default 16-bit files it rips into 24-bit files? Any help would be appreciated.

Thank you!

**Comment by enzo1982** [comment]
Hi, the native format of audio CDs is 16 bit, so there is usually no use in converting them to 24 bit.

The only exception may be HDCDs which can be decoded to 24 bit of which 20 bit are effectively used. You can do this by enabling the HDCD decoder in fre:ac's processing settings. Ripping with the HDCD decoder enabled will only output 24 bit samples for actual HDCDs. If no HDCD encoding is detected, the decoder will leave the samples unchanged at 16 bit.

However, as you already have 16 bit FLAC files of your CDs, you can just run those through the HDCD decoder. No need to re-rip the actual CDs.

HDCDs are a mixed bag, though. There are many "fake HDCDs" which signal the presence of HDCD encoding, but don't actually make use of the additional dynamic range. This results in the HDCD decoder just blowing up the samples to 24 bit and reducing the volume by 6db without any actual audio quality gain.
